homeassistant/components/eafm/sensor.py

```diff
@@ -96,6 +99,7 @@ def __init__(self, coordinator, key):
         """Initialise the gauge with a data instance and station."""
         super().__init__(coordinator)
         self.key = key
+        self._attr_state_class = SensorStateClass.MEASUREMENT
```

Comment:
For attributes that don't change, you should set this on the class. E.g. put it below the `attribution = ` line (and then you won't need the self either).

Reply:
Ok that is now done